backend/Functions/Edna.Connect/Lti1Api.cs

```diff
@@ -78,10 +78,23 @@ private Assignment ConvertRequestToAssignment(LtiRequest ltiRequest)
                 ResourceLinkId = ltiRequest.ResourceLinkId,
                 Name = ltiRequest.ResourceLinkTitle,
                 CourseName = ltiRequest.ContextTitle,
-                LtiVersion = ltiRequest.Version,
+                LtiVersion = GetLtiVersion(ltiRequest),
                 ContextMembershipsUrl = ltiRequest.Parameters.FirstOrDefault(pair => pair.Key == "custom_context_memberships_url").Value,
                 OAuthConsumerKey = ltiRequest.ConsumerKey
             };
         }
+
+        private static string GetLtiVersion(LtiRequest ltiRequest)
+        {
+            switch(ltiRequest.Version)
+            {
+                case "1.0":
+                    return LtiVersion.Lti1.ToString();
+                case "1.3.0":
+                    return LtiVersion.LtiAdvantage.ToString();
+                default:
+                    return null;

```

Comment:
Should we log this information so that we at least know what is getting passed to us at this location 🤔 ?